### PR TITLE
[WIP] Fix doc

### PR DESCRIPTION
**Pull request by kitchoi**
Some fixes for the doc

### CODE REVIEW

File: simphony_mayavi/sources/cuds_source.py [hunk 1]
```diff
@@ -98,8 +98,9 @@ def __init__(self, cuds=None, point_scalars=None, point_vectors=None,
 
         Parameters
         ----------
-        cuds : ABCParticles, ABCLattice, ABCMesh or H5Mesh
+        cuds : Instance
```
Comment (kitchoi) [inline simphony_mayavi/sources/cuds_source.py:88]: @itziakos  bug in sectiondoc?

**Comment by kitchoi** [comment]
partial fix to #172

**Comment by kitchoi** [comment]
Somehow the "#: " syntax does not work for `EngineSource`
http://simphony-mayavi.readthedocs.org/en/fix-doc/api/sources.html#simphony_mayavi.sources.engine_source.EngineSource

**Comment by stefanoborini** [comment]
:+1: Merging

**Pull request merged**